api/store.js

```diff
@@ -1,172 +0,0 @@
-const fs = require('fs');
```

Comment:
This seems possibly bigger than just the images, so I just wanted to flag to make sure we want to delete this?

Reply:
It was originally intended for uploading RFPs and contracts, but we never used it. The upload API endpoints in this PR can be used for the RFPs and contracts too, I'm pretty sure, so no need for this parallel implementation!